fabtests/multinode/src/core_coll.c

```diff
@@ -278,7 +278,8 @@ static int all_gather_test_run()
 			goto errout;
 		}
 	}
-	return FI_SUCCESS;
+
+	err = FI_SUCCESS;
```

Comment:
Please rename 'err' to ret and 'errout' below to out, since neither are no longer error cases.